src/presets/erc721.cairo

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts for Cairo v0.8.0-beta.0 (presets/erc721.cairo)
+
+/// # ERC721 Preset
+///
+/// The ERC721 contract offers a batch-mint mechanism that
+/// can only be executed once upon contract construction.
+#[starknet::contract]
+mod ERC721 {
+    use openzeppelin::introspection::src5::SRC5Component;
+    use openzeppelin::token::erc721::ERC721Component;
+    use starknet::ContractAddress;
+
+    component!(path: SRC5Component, storage: src5, event: SRC5Event);
+    component!(path: ERC721Component, storage: erc721, event: ERC721Event);
```

Comment:
I think we should swap the components to make ERC721 appear first (this applies to impls below too). That is consistent with the Account Preset as well.

Reply:
I agree in the sense that it's the "main" component, but I switched it because `Access` appears first in the imports and IMO it appears more organized that way. What do you think?

Reply:
mmm, got your point, but I still think it makes more sense to have the main component first, instead of letting the imports decide. We still need to define the rule for this, but I don't like the one coming from import orders that much 😢

Reply:
That's fair, and we also already have the account preset using the suggested approach. I'll update and we can maybe revisit in the future

Reply:
i actually like `SRC5` first, as it's a dependency of `ERC721`. also, not sure if there's always going to be a "main component", and it's not clear to me that's a concept we want to suggest or push.

Reply:
> also, not sure if there's always going to be a "main component", and it's not clear to me that's a concept we want to suggest or push.

> +1 to not inheriting order from imports

Let's say we create an ownable, pausable ERC20 preset. There's no component dependency like with SRC5. If we're not using the main component approach and we're not inheriting order from imports, how else can we organize the component declarations?

Reply:
maybe we don't need a rule for that